examples/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ set(GPGPU_GLES_EXAMPLES
     array_add_float
     fir_conv_float
     array_add_fixed16
+    mult_mat_int
```

Comment:
why not floats? ints make little sense here

Reply:
We have changed the entire operation on ints to that on floats

Reply:
so change the name to float please :)

Reply:
Yes, done.